src/tests/eosjs-jsonrpc.test.ts

```diff
@@ -606,6 +606,47 @@ describe('JSON RPC', () => {
         expect(fetch).toBeCalledWith(endpoint + expPath, expParams);
     });
 
+    it('calls send_readonly_transaction', async () => {
+        const expPath = '/v1/chain/send_readonly_transaction';
+        const signatures = [
+            'George Washington',
+            'John Hancock',
+            'Abraham Lincoln',
+        ];
+        const serializedTransaction = new Uint8Array([
+            0, 16, 32, 128, 255,
+        ]);
+
+        const limit = 50;
+        const expReturn = {
+            data: '12345',
+            transaction_id: '54321',
+            processed: {
+                block_num: 671823
+            }
+        };
+        const callParams = {
```

Comment:
We should test to make sure serializedContextFreeData is being used as expected in the method.